File: python/kode program/Store.py
from Electronic import Electronic

# Store mewarisi dari Electronic
class Store(Electronic):
    def __init__(self, id, name, price, stock, image,
                 brand, warranty, category,
                 location, seller_name, contact):
        # panggil constructor Electronic
        super().__init__(id, name, price, stock, image, brand, warranty, category)
        self._location = location
        self._seller_name = seller_name
        self._contact = contact

    # setter dan getter untuk location
    def set_location(self, location):
        self._location = location

    def get_location(self):
        return self._location

    # setter dan getter untuk seller_name
    def set_seller_name(self, seller_name):
        self._seller_name = seller_name

    def get_seller_name(self):
        return self._seller_name

    # setter dan getter untuk contact
    def set_contact(self, contact):
        self._contact = contact

    def get_contact(self):
        return self._contact

File: java/kode program/Store.java
public class Store extends Electronic {
    private String location;
    private String sellerName;
    private String contact;

    public Store(int id, String name, int price, int stock, String image,
                 String brand, String warranty, String category,
                 String location, String sellerName, String contact) {
        super(id, name, price, stock, image, brand, warranty, category);
        this.location = location;
        this.sellerName = sellerName;
        this.contact = contact;
    }

    // setter dan getter untuk location
    public void setLocation(String location) {
        this.location = location;
    }
    public String getLocation() {
        return this.location;
    }

    // setter dan getter untuk sellerName
    public void setSellerName(String sellerName) {
        this.sellerName = sellerName;
    }
    public String getSellerName() {
        return this.sellerName;
    }

    // setter dan getter untuk contact
    public void setContact(String contact) {
        this.contact = contact;
    }
    public String getContact() {
        return this.contact;
    }
    
}
